src/@batch-flask/core/aad/access-token-cache/access-token-cache.spec.ts

```diff
@@ -5,13 +5,12 @@ import { AccessTokenCache } from "./access-token-cache";
 const tenant1 = "tenant-1";
 const resource1 = "http://example.com";
 const token1 = {
-    access_token: "sometoken",
-    expires_on: DateTime.local().plus({ hours: 1 }).toJSDate(),
+    accessToken: "sometoken",
```

Comment:
Nit: Why are some of these updated to camel case but others are left as snake case?